libraries/c_sdk/standard/mqtt/include/mqtt_config.h

```diff
@@ -22,6 +22,13 @@
 
 /************ End of logging configuration ****************/
 
+/* Set network context used for sending the packets on the network. */
+typedef struct NetworkContext
```

Comment:
This structure definition can be internal to the Shim. MQTT library already have the declaration for the structure in transport_interface.h  MQTT library doesn't need to know the internal details about the NetworkContext.